app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,292 @@
+    <string name="help_permission1_heading">Camera</string>
+    <string name="help_privacy_heading">Welke machtigingen vereist de PRIVACYVRIENDELIJKE QR-scanner?</string>
+    <string name="help_category_example">Het scannen werkt niet. Wat kan ik hieraan doen?</string>
+    <string name="help_overview_heading">Wat is de PRIVACYVRIENDELIJKE QR-scanner?</string>
+    <string name="help_intro">Met de PRIVACYVRIENDELIJKE QR-codescanner-app kun je barcodes en QR-codes met vari?rende inhoud scannen.</string>
+    <string name="help_whatis_answer">Met de PRIVACYVRIENDELIJKE QR-codescanner-app kun je barcodes en QR-codes met vari?rende inhoud scannen en genereren. Daarnaast kun je gescande QR-codes kopi?ren naar of delen met andere apps. Ook kun je gegeneerde QR-codes opslaan in de galerij en delen.</string>
+
+
+    <!--###HELP###-->
+
+
+    <string name="help_whatis">Wat is de PRIVACYVRIENDELIJKE QR-scanner?</string>
+
+    <string name="help_usability">Het scannen werkt niet. Wat kan ik hieraan doen?</string>
+    <string name="help_usability_answer">Als je geen codes kunt scannen, vergroot dan de afstand tussen je apparaat en de code.</string>
+    <string name="help_privacy">Wat maakt de app privacyvriendelijk?</string>
+    <string name="help_privacy_answer">Privacy Friendly App-voorbeeld vereist geen enkele machtiging. Daarnaast wordt er niks bijgehouden en geen reclame getoond.</string>
+    <string name="help_permission">Welke machtigingen vereist de PRIVACYVRIENDELIJKE QR-scanner?</string>
+    <string name="help_permission_answer">Deze app vereist de cameramachtiging om barcodes and QR-codes te kunnen scannen middels de camera.</string>
+    <!-- Strings related to SettingsActivity -->
+
+    <!-- Example General settings -->
+    <string name="pref_title_enable_history">Geschiedenis bijhouden</string>
+    <string name="pref_description_enable_history">Sla eerdere scans op zodat je ze later weer kunt bekijken</string>
+    <string name="pref_keep_duplicates">Duplicaten bijhouden</string>
+    <string name="pref_keep_duplicates_explanation">Neem duplicaten op in de geschiedenis</string>
+
+    <!-- Example settings for Notifications -->
+    <string name="pref_title_beep">Geluid</string>
+    <string name="description_Mms">De gescande QR-code bevat een mms.</string>
+    <string name="geoInfo">De gescande QR-code bevat geografische informatie.</string>
+    <string name="content_copied">Inhoud gekopieerd</string>
+    <string name="share_via">Delen via</string>
+    <string name="no_content_in_history">Er is geen geschiedenis!</string>
+    <string name="selected_item">item geselecteerd</string>
+    <string name="selected_items">items geselecteerd</string>
+    <string name="removed_item">item verwijderd</string>
+    <string name="removed_items">items verwijderd</string>
+    <string name="canncelled_scan">Het scannen is afgebroken</string>
+    <string name="Scan_qr">Scannen</string>
+    <string name="okay">Ok?</string>
```

Comment:
Why is there a question mark?

Reply:
It was supposed to be an 'é', but somehow GitHub messed up the formatting during the push. Thanks for noticing, just fixed it! :)